src/AccountInfoClient.php

```diff
@@ -0,0 +1,94 @@
+<?php
+declare(strict_types=1);
+
+namespace JVDS\YukiClient;
+
+use JVDS\YukiClient\Exception\Exception;
+use JVDS\YukiClient\Exception\UnexpectedTypeException;
+use JVDS\YukiClient\Soap\ClientFactory;
+use SimpleXMLElement;
+
+class AccountInfoClient extends Client
+{
+    private const WSDL = 'https://api.yukiworks.nl/ws/AccountingInfo.asmx?wsdl';
+
+    protected const SESSION_ID_ARG = 'sessionID';
+
+    public function __construct(
+        string $accessKey,
+        array $soapClientOptions = [],
+        ClientFactory $factory = null,
+        string $wsdl = self::WSDL
+    ) {
+        parent::__construct($accessKey, $wsdl, $soapClientOptions, $factory);
+    }
+
+
+    public function getTransactionDetails(
+        string $administrationID,
+        string $GLAccountCode = "",

```

Comment:
Could you use single quotes here, to keep it consistent?
```suggestion
        string $GLAccountCode = '',
```